### DISCUSSION TITLE
Adstock Rule-Of-Thumb Ranges

### DISCUSSION THREAD

**Discussion by GrizzledLotus** [facebookexperimental/Robyn#358] [Q&A]
You state the following about adstock ranges: "Rule-of-thumb for common media genre: TV c(0.3, 0.8), OOH/Print/Radio c(0.1, 0.4), digital c(0, 0.3)". Where do these typical ranges come from? I'm trying to convince my higher-ups in using the Robyn approach but they have reservations. One of them being the source of these "industry standards".  Are these ranges dependent on the type of business? MMM had its start with Consumer Packaged Goods and I'm trying to give them evidence that these ranges are valid for our business situation. Thanks in advance.

**Comment by kyletgoldberg** [comment]
Hi - these ranges are based mostly on our own experience as modelers having seen and built many models, that said these ranges are based on weekly data, and if there is hesitancy to use them it is totally reasonable to widen the ranges and let the hyperparameter optimization go to work. There's no need to strictly adhere to them, they're just guidance as a starting point since many users were not sure what a good starting point would be.

**Reply by CJ2407** [reply]
@kyletgoldberg You have mentioned above that the ranges are recommended for weekly data. Do you have some range recommendation for Weibull PDF for the daily data?
